pinning.go

```diff
@@ -0,0 +1,13 @@
+package config
+
+// Pinning configures the pinning services.
+type Pinning struct {
+	// Services lists the pinning services
+	Services []PinningServices
```

Comment:
@aschmahmann @jacobheun  Q: How are we going to store automated policy per Pinning service?

IIUC we will have the default which is `null` or  explicit `manual` and there will be opt-in `mfs` which enables remote pin on every MFS root change.

Do we need to add `PinningService.Policy`?

Reply:
Bump @aschmahmann  @petar

Reply:
I believe we need array. For now it will be either "manual" or "all in MFS", but I can imagine the future when one can have both MFS and local pins mirrored to remote service:
```
Policies: ["mfs-root", "local-pins"]
```

Reply:
Yep, I think there should be an array of policies per service. I think @petar wasn't sure if we wanted Pinning.RemoteServices to be an array or a map indexed by the service name.